### ISSUE TITLE
Option to run covr instead of the regular unit testing workflow

### ISSUE THREAD

**Issue by wlandau-lilly** [r-lib/rcmdcheck#57]
When I build in Travis-CI, I run the regular package checks and then run `covr::package_coverage()` after success. My unit tests are long, so there is a lot of duplicated effort. If `covr` runs and reports coverage during the "checking tests" phase itself, build time will drop from 10 minutes to 5 minutes for my main use case. Is it possible to do this in `rcmdcheck` and still have a report to upload to [codecov.io](https://codecov.io)?

**Comment by gaborcsardi** [comment]
No, not possible, sorry. `rcmdcheck` is concerned about running `R CMD check`, and `covr` does not do that.

**Comment by wlandau-lilly** [comment]
@jimhester's suggestion [here](https://github.com/travis-ci/travis-ci/issues/8864#issuecomment-349684435) does what I need.
